client/src/components/GithubLogin.tsx

```diff
@@ -38,7 +38,7 @@ export const GithubLogin = ({ deviceCode }: { deviceCode?: string }) => {
 
     const params = new URLSearchParams({
       client_id: import.meta.env.VITE_GITHUB_CLIENT_ID,
-      redirect_uri: import.meta.env.VITE_GITHUB_REDIRECT_URI,
+      redirect_uri: `${import.meta.env.VITE_API_SERVER}/auth/callback`,
```

Comment:
wait so can you clarify where the VITE environment is getting set? I thought we agreed that the environment for the static client should be loaded from the server?

Reply:
as discussed, this will be done at a later pr. for now, it is getting set in your .zshrc env